tcms/testcases/tests/test_models.py

```diff
@@ -157,3 +159,21 @@ def test_send_mail_to_case_author(self, send_mail):
             recipients,
             fail_silently=False,
         )
+
+
+class TestActualDurationProperty(BasePlanCase):
+    """Test TestExecution.actual_duration"""
+
+    @classmethod
+    def setUpTestData(cls):
+        super(TestActualDurationProperty, cls).setUpTestData()
+
+        cls.test_execution = TestExecution()
+        cls.test_execution.start_date = timezone.now()
+        cls.test_execution.stop_date = timezone.now() + timezone.timedelta(days=10)
+
+    def test_create_actual_duration(self):
```

Comment:
Naming of test method should reflect the actual behavior that is being checked to make it obvious what is happening. 

That said you have 3 behaviors:
1) start & stop date both have a timestamp value and we want to check that the calculation gives the expected timedelta
2) start_date is None (see null=True, blank=True in the model definition) aka test execution hasn't been started yet
3) stop_date is None aka test execution hasn't been finished yet

For 2) the expected result is None because you can't have a duration if you don't have a beginning. 

For 3) I am not sure ATM, either None or timezone.now() - start_date (e.g. duration from start up to now!). Pick one of them and hard-code it in a test, we'll adjust it later once we figure out how we want to use this property in the rest of the application.